### PR TITLE
Add missing functions to jax.numpy docs

### PR DESCRIPTION
**Pull request by jlebar**
I structured this as two commits -- first, sort the original list, then add the new elements -- so that you can confirm I'm not deleting anything.

### CODE REVIEW

**Comment by hawkinsp** [comment]
It looks like your "how to regenerate this list" code snippet is falling afoul of our documentation testing.

**Comment by jakevdp** [comment]
I think if you add `   # doctest: +SKIP` at the end of the snippet, it will fix the CI error.

File: docs/jax.numpy.rst [hunk 1]
```diff
@@ -29,6 +29,14 @@ input data.
 
 Not every function in NumPy is implemented; contributions are welcome!
 
+.. Generate the list below as follows:
+   >>> import jax.numpy, numpy
+   >>> fns = set(dir(numpy)) & set(dir(jax.numpy)) - set(jax.numpy._NOT_IMPLEMENTED)
+   >>> print('\n'.join('    ' + x for x in fns if callable(getattr(jax.numpy, x))))
```
Comment (jakevdp) [inline docs/jax.numpy.rst:35]: Add `# doctest: +SKIP` at the end of this line.

Comment (jlebar) [reply docs/jax.numpy.rst]: Fixed, thanks!

**Comment by jakevdp** [comment]
Looks good, thanks!

**Pull request merged**